lib/charms/vault_k8s/v0/juju_facade.py

```diff
@@ -545,6 +549,15 @@ def get_storage_location(self, storage_name: str) -> Path:
             raise NoSuchStorageError(f"Storage {storage_name} not found")
         return storages[storage_name][0].location
 
+    def get_ingress_address(self, relation) -> str | None:
```

Comment:
We're missing the input type

Reply:
https://github.com/canonical/vault-k8s-operator/pull/534/commits/6e750f519fc9a3f086e72f8c72290465fd010926